annotations/views/citesphere_views.py

```diff
@@ -106,4 +109,152 @@ def list(self, request, repository_pk, groups_pk):
         manager = repository.manager(request.user)
         page = request.query_params.get('page', 1)
         items = manager.group_items(groups_pk, page)
-        return Response(items)
\ No newline at end of file
+        return Response(items)
+    
+    def retrieve(self, request, repository_pk, groups_pk, pk):
+        part_of_id = self.request.query_params.get('part_of', None)            
+        found, project_details, part_of_project = utils._get_project_details(
+            request,
+            repository_pk
+        )
+        if not found:
+            return Response({ "message": f"Project not found" }, 404)
+        repository = get_object_or_404(
+            Repository,
+            pk=repository_pk,
+            repo_type=Repository.CITESPHERE
+        )
+        manager = repository.manager(request.user)
+        item_data = manager.group_item(groups_pk, pk)
+        try:
+            if item_data[0] == "error":
+                return Response(status=item_data[1])
+        except Exception as e:
+            pass
+        data = item_data['item']['gilesUploads']
+        result = data[2]
```

Comment:
Accessing one of the files, since in vogon we need to display only a single file